airflow/models/taskinstance.py

```diff
@@ -1004,6 +1004,7 @@ def signal_handler(signum, frame):
             self.state = State.SUCCESS
         except AirflowSkipException as e:
             # Recording SKIP
+            # This change is in reference to [AIRFLOW-5653][CX-16266]
```

Comment:
do we need this statement?

Reply:
This commit was done by me in OS Airflow 1.10.6 so it need not be cherry picked. That should remove this comment as well.